src/connector/src/parser/debezium/debezium_parser.rs

```diff
@@ -33,6 +35,27 @@ pub struct DebeziumParser {
     payload_builder: AccessBuilderImpl,
     pub(crate) rw_columns: Vec<SourceColumnDesc>,
     source_ctx: SourceContextRef,
+
+    props: DebeziumProps,
+}
+
+pub const DEBEZIUM_IGNORE_KEY: &str = "ignore_key";
+
+#[derive(Debug, Clone, Default)]
+pub struct DebeziumProps {
+    // Ignore the key part of the message.
+    // If enabled, we don't take the key part into message accessor.
+    pub ignore_key: bool,
+}
+
+impl DebeziumProps {
+    pub fn from(props: &BTreeMap<String, String>) -> Self {
+        let ignore_key = props
+            .get(DEBEZIUM_IGNORE_KEY)
+            .map(|v| v.eq_ignore_ascii_case("true"))
+            .unwrap_or(false);
+        Self { ignore_key }
+    }
```

Comment:
Also note that `format_endcode_options` are not tracked in with option checks (of course they are not in with clause). 
I am wondering if we need another tool to track this options